src/rad/resources/schemas/l1_detector_guidewindow-1.0.0.yaml

```
@@ -0,0 +1,1408 @@
+      sci_data_packet_counts:
+        title: Science Data Packet Counts
+        description: |
+          2D array of the science data packet counts for the matched
+          pedestal and signal resultants. The span between the pedestal
+          and the signal values indicate which full frame row indices
+          were read out during the guide window exposure.
+        ndim: 2
+        datatype: int16
+        exact_datatype: true
+      sce_status_flag:
+        title: SCE Status Flag
+        description: |
+          Either a 2D array containing the SCE status for each pedestal
+          and signal resultant or a single integer if all flags are equal.
+          1 indicates an internal error occurred during the ACADIA processing.
+        anyOf:
+          - type: integer
+          - tag: tag:stsci.edu:asdf/core/ndarray-1.*
+            datatype: int16
+            exact_datatype: true
+            ndim: 2
+      gw_resultant_err_flag:
+        title: Guide Window Resultant Error Flag
+        description: |
+          Either a 2D array containing the guide window resultant error flag
+          for each pedestal and signal resultant or a single integer if all
+          flags are equal. 1 indicates an internal error in the SCU occurred
+          while processing the GW resultant. See FPGA error registers for
+          more information.
+        anyOf:
+          - type: integer
+          - tag: tag:stsci.edu:asdf/core/ndarray-1.*
+            datatype: int16
+            exact_datatype: true
+            ndim: 2
+      min_bind_flag:
+        title: Minimum Binding Flag
+        description: |
+          Either a 2D array containing the binding minimum flag for each pedestal
+          and signal resultant or a single integer if all flags are equal.
+          1 indicates that any negative pixel values within the resultant were bound to 0.
+        anyOf:
+          - type: integer
+          - tag: tag:stsci.edu:asdf/core/ndarray-1.*
+            datatype: int16
+            exact_datatype: true
+            ndim: 2
+      max_bind_flag:
+        title: Maximum Binding Flag
+        description: |
+          Either a 2D array containing the binding maximum flag for each pedestal
+          and signal resultant or a single integer if all flags are equal.
+          1 indicates that any positive pixel values within the resultant were bound to 65535.
+        anyOf:
+          - type: integer
+          - tag: tag:stsci.edu:asdf/core/ndarray-1.*
+            datatype: int16
+            exact_datatype: true
+            ndim: 2
+      # Optional
+      exposure_mapping:
+        title: Exposure Mapping
+        description: |
+          1D array of signed integers with the same length as the tracking signal resultants.
+          Positive integers correspond to the exposure number (single MA Table read).
+          Negative integers correspond to the tracking reads between exposures.
+        ndim: 1
```

```suggestion
        tag: tag:stsci.edu:asdf/core/ndarray-1.*
        ndim: 1
```
Missing `tag` keyword.